assets/src/scss/layout/_block-classes.scss

```diff
@@ -1,26 +1,79 @@
 .wp-block-query-pagination {
+  a,
+  .page-numbers {
+    color: $black;
+    padding: $sp-1 $sp-1x;
+    user-select: none;
+    border: 1px solid transparent;
+    border-radius: 4px;
+  }
+
   a {
-    color: black;
-    font-weight: bold;
+    font-weight: 500;
+
+    &:hover {
+      text-decoration: none;
+      background-color: $grey-10;
+      transition-property: background-color, border-color;
+      transition-duration: 250ms;
+      transition-timing-function: linear;
+    }
   }
 
   font-family: $roboto;
   display: flex;
   flex-direction: row;
-  justify-content: center;
+  justify-content: space-between;
   width: 100%;
-  margin-bottom: 16px;
+  margin-bottom: $sp-2;
+
+  @include large-and-up {
+    justify-content: center;
+  }
+}
+
+.wp-block-query-pagination-previous,
+.wp-block-query-pagination-next {
+  margin: 0 !important;
+}
+
+.wp-block-query-pagination-previous:before,
+.wp-block-query-pagination-next:after {
+  content: "";
+  height: 12px;
+  width: 8px;
+  display: inline-block;
+  mask-image: url("../../images/chevron.svg");
+  mask-repeat: no-repeat;
+  mask-size: contain;
+  mask-position: center;
+  background-repeat: no-repeat;
+  background-color: currentColor;
+  color: $black;
 }
 
-.page-numbers {
-  &:not(:last-child) {
-    margin-inline-end: 8px;
+.wp-block-query-pagination-previous {
+  &:before {
+    margin-right: 6px;
+    transform: rotate(-180deg);
   }
+}
 
-  &.current {
-    border: 1px solid black;
-    border-radius: 4px;
-    padding: 8px 12px;
+.wp-block-query-pagination-next {
+  &:after {
+    margin-left: 6px;
```

Comment:
This should be `margin-inline-start` to be compatible with RTL sites

Reply:
Same here!